recipe-server/client/control_new/components/common/FormActions.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+
+/**
+ * Container for buttons displayed at the bottom of a form.
+ *
+ * Example:
+ *
+ * <FormActions>
+ *   <FormActions.Primary>
+ *     <Button>Save</Button>
+ *   </FormActions.Primary>
+ *   <FormActions.Secondary>
+ *     <Button>Delete</Button>
+ *   </FormActions.Secondary>
+ * </FormActions>
+ */
+export default function FormActions({ children }) {
```

Comment:
I didn't plan it, but I figured that we'd be adding action button containers to enough forms that abstracting it would be useful. If we decide in the future to hide secondary actions in a collapsible container or something, we don't have to update the container since it's purely semantic.

I can remove it if you think it's overkill without the planned extra logic.